cmake/Modules/Findportaudio.cmake

```diff
@@ -0,0 +1,42 @@
+# portaudio_FOUND: whether portaudio was found
+# portaudio_static: imported module
+
+include(FindPackageHandleStandardArgs)
+
+find_path(portaudio_INCLUDE_DIR
+	NAMES
+		portaudio.h
+	PATHS
+		/usr/include
+		/usr/local/include
+		/opt/local/include
+		/sw/include
+)
+
+find_library(portaudio_LIBRARY
+	NAMES
+		portaudio portaudio_x64
+	PATHS
+		/usr/lib
+		/usr/local/lib
+		/opt/local/lib
+		/sw/lib
```

Comment:
these hardcoded paths will fail on many linux systems, maybe it should be combined with inferred search paths from the operating system's environment before checking hard-coded paths?

Reply:
I think cmake should do that automatically? https://cmake.org/cmake/help/latest/command/find_library.html

Reply:
Yeah these are additional hints but cmake will check the normal locations dependning on the platform. Maybe use the word HINTS instead of PATHS as I was unsure until I checked. It seems like they do the same thing?